build.go

```diff
@@ -68,6 +71,139 @@ func if_not_exists(cmd string, cb func()) {
 	}
 }
 
+func removeExt(s string) string {
+	return s[:len(s)-len(filepath.Ext(s))]
+}
+
+func unzip(src, dest, fileToExtract string) error {
+	r, err := zip.OpenReader(src)
+    if err != nil {
+        return err
+    }
+    defer func() {
+        if err := r.Close(); err != nil {
+            panic(err)
+        }
+    }()
+
+    os.MkdirAll(dest, 0755)
+
+    // Closure to address file descriptors issue with all the deferred .Close() methods
+    extractAndWriteFile := func(f *zip.File) error {
+       rc, err := f.Open()
+        if err != nil {
+            return err
+        }
+        defer func() {
+            if err := rc.Close(); err != nil {
+                panic(err)
```

Comment:
I was under the impression that panic was something to be used sparingly in go. It also seems this code doesn't intend to recover from this panic further up the stack. Why choose it instead of printing the error and exiting?